jekyll/_cci2/runner-installation.adoc

```diff
@@ -194,6 +194,26 @@ mkdir -p /opt/circleci/workdir
 chown -R circleci /opt/circleci/workdir
 ```
 
+=== Configure SELinux policy (RHEL 8)
+
+An SELinux policy is required for runner to accept and launch jobs on RHEL 8 systems. Note that this policy does not add any permissions to the ones that may be required by individual jobs on this runner install.
+
+Create directory `/opt/circleci/policy`, owned by `root`, and generate the initial policy module:
+
+```bash
+sudo install -d -o root /opt/circleci/policy
+
+sepolicy generate --path /opt/circleci/policy --init /opt/circleci/circleci-launch-agent
+```
+
+Copy the following type enforcing file https://raw.githubusercontent.com/CircleCI-Public/runner-installation-files/main/rhel-install/circleci_launch_agent.te[`circleci_launch_agent.te`] to `/opt/circleci/policy/circleci_launch_agent.te`, owned by `root`:
```

Comment:
Will we eventually need TE files for other RHEL releases, such that we should put an "8" somewhere in this path, or does the single file support multiple versions via some other mechanism, a la "#if" ?

Reply:
Hi Steve, I checked on another version of RHEL (7) and discovered that this TE file is _not_ compatible with that release. I added an 8 to the path so we can support other versions in the future. Thanks for catching that.